core/src/main/java/org/frankframework/receivers/Receiver.java

```diff
@@ -1169,7 +1176,10 @@ public void retryMessage(String storageKey) throws ListenerException {
 	private Message processMessageInAdapter(MessageWrapper<M> messageWrapper, PipeLineSession session, long waitingDuration, boolean manualRetry, boolean duplicatesAlreadyChecked) throws ListenerException {
 		final long startProcessingTimestamp = System.currentTimeMillis();
 		final String logPrefix = getLogPrefix();
-		try (final CloseableThreadContext.Instance ignored = LogUtil.getThreadContext(getAdapter(), messageWrapper.getId(), session)) {
+		try (final CloseableThreadContext.Instance ignored = LogUtil.getThreadContext(getAdapter(), messageWrapper.getId(), session);
+			 final IbisMaskingLayout.HideRegexContext ignored2 = IbisMaskingLayout.pushToThreadLocalReplace(hideRegexPattern);
+			 final IbisMaskingLayout.HideRegexContext ignored3 = IbisMaskingLayout.pushToThreadLocalReplace(getAdapter().getComposedHideRegexPattern());
```

Comment:
De adapter voegt toch al haar eigen composed pattern toe?

Reply:
Niet via de method die de receiver gebruikt. Ik besloot dat hier in de receiver te doen voor de adapter, zodat er meer log-statements zijn waarop deze hide-regexes toegepast worden. Ik kan in de tests zien dat dit verschil maakt met wanneer ik de hide-regex van de Adapter pas toepas in de method `processMessageWithExceptions`, daarom dat ik het heb verplaatst hiernaar.